docs/how-to-docs/cert-manager-upgrade-from-v0.8.md

```diff
@@ -0,0 +1,347 @@
+  - amend the `Certificate`'s annotations and labels
+  - change the value of `secretName`
+  - amend your deployment to mount the new secret (e.g. get the nginx sidecar to mount the new tls secret)
+- Deploy the changes
+- When you've checked that the service is functioning as intended, you can tidy up the old v0.8 cert-manager resources:
+  - delete any certficate resources still returned by `kubectl -n project get certificate` (back them up if they are not stored in git)
+  - delete any secrets associated with those old resources (again, back them up to be safe)
+- You can check that the certificate resources are valid by running `kubectl -n project get certificate.cert-manager.io`. The `READY` field for the resources should be `TRUE`. Note that it might take a short while (typically no more than about a minute) for the certificates to reach that `READY` state.
+
+The main draw-back of this approach is that the value for the new secret being created will need to be created from LetsEncrypt (unless using the platform culster issuer). This is usually quite quick, but could take up to around 1 minute.
+
+If your deployments have several replicas and use a rolling update strategy, as they should, this is not usually an issue as the service will continue being available, although with a reduced capacity. As the deployment roll-out, users might experience a failed request that should be successful again on retry.
+
+### Option 2 - keeping same secret names
+
+This option is much more complex than the option 1 and should only be considered if there are concerens with service availability. For example, if there is a single replica and a deployment would create an unacceptable service outage while a new certificate is retrieved. But then again, you're not running services with a single replica as that' a bad pattern, right?
+
+Please be aware that if not performed appropriately, this upgrade path has the potential to hit LetsEncrypt limits and therefore prevent new certficates to be successfully retrieved for a given hostname for up to a week.
+
+- For `Ingress` resources:
+  - amend the `Ingress`'s annotations and labels to **REMOVE** all `certmanager.k8s.io` annotations. **DO NOT** add any of the new cert-manager annotations yet.
+- Deploy the ingress changes. This will unregister the ingresses from the old version of cert-manager and within a few minutes, the `Certificate` resources that cert-manager created automatically will be deleted. The secrets with the tls private key and certificate will still be available and therefore your ingress will carry on working until that certficate expires. That's because, by default, cert-manager does not delete the secrets specified in a certificate object when that certificate object is deleted. You typically have several weeks during which the certifcate will remain valid.
```

Comment:
`certficate` => `certificate` (maybe do a quick search replace in the doc as there may be a few occurrences of this)